website/versioned_docs/version-0.11.0/cli.md

```diff
@@ -5,7 +5,7 @@ last_modified_at: 2021-08-18T15:59:57-04:00
 ---
 
 ### Local set up
-Once hudi has been built, the shell can be fired by via  `cd hudi-cli && ./hudi-cli.sh`. A hudi table resides on DFS, in a location referred to as the `basePath` and
+Once hudi has been built, the shell can be fired by via  `export CLIENT_JAR=hudi-cli/target/hudi-cli-xx.jar && cd hudi-cli && ./hudi-cli.sh`. A hudi table resides on DFS, in a location referred to as the `basePath` and
```

Comment:
there is no need to set extra command in the guide. the `hudi-cli.sh` already prompt users to set `hudi-env.sh` so people should just change it here, instead of having inline env var, which does not look clean